CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(BoostMp11 VERSION 1.68.0 LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+include(CTest)
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+
+add_library(mp11 INTERFACE)
+add_library(Boost::mp11 ALIAS mp11)
+
+target_include_directories(mp11 INTERFACE
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
+)
+
+if (BUILD_TESTING)
+    find_package(Boost 1.56 REQUIRED)  # 1.56+ for lightweight_test
```

Comment:
Travis of course has 1.55. FetchContent is best, but that's CMake 3.11, and Travis has 3.9. It ain't easy.

Reply:
Ordinarily I'd just submodule Config/Core/Assert, but it's not appropriate for a Boost library to do that.